client/src/reducers/controls.ts

```diff
@@ -91,7 +91,7 @@ const Controls = (
       loading: false,
     },
     imageOpacity: isTest ? 0 : 100,
-    dotOpacity: 50,
+    dotOpacity: 100,
```

Comment:
As per Harley - instead of setting default opacity, update the default color to dark gray for spatial to work with mix blend mode effect